kiwi/container/docker.py

```diff
@@ -113,12 +116,24 @@ def create(self, filename):
             [container_dir, self.container_tag]
         )
 
-        Command.run(
-            ['umoci', 'init', '--layout', container_dir]
-        )
-        Command.run(
-            ['umoci', 'new', '--image', container_name]
-        )
+        if base_image:
+            uncompressor = Compress(base_image)
+            uncompressor.uncompress(True)
+            self.uncompressed_image = uncompressor.uncompressed_filename
+
+            Command.run([
+                'skopeo', 'copy',
+                'docker-archive:{0}'.format(self.uncompressed_image),
+                'oci:{0}'.format(container_name)
+            ])
```

Comment:
I don't understand why this step is needed ? iirc we unpack the base image, then the base image is copied as file into this unpacked directory and then it is uncompressed and turned into an oci image. Is that correct, and if so I'm clueless why we do that :) can you enlight me ?